pkg/handler/handler.go

```diff
@@ -167,6 +168,9 @@ func (o *OpenAPIService) RegisterOpenAPIVersionedService(servePath string, handl
 					if clause.SubType != accepts.SubType && clause.SubType != "*" {
 						continue
 					}
+					if accepts.SubType == "com.github.proto-openapi.spec.v2@v1.0+protobuf" {
+						klog.Info("Deprecated: use SubType com.github.proto-openapi.spec.v2.v1.0+protobuf instead.")
```

Comment:
I wouldn't bother with this... this log message isn't going to go anywhere the caller will see it

Reply:
@liggitt This message would not help client should we delete this. Any idea around how to communicate the message with caller ?  Thank you.

Reply:
We'll probably have to support this forever, which is fine, if we keep using this and return the new thing, we're good to go IMO. You can go ahead and remove the warning.